pkg/clusterversion/cockroach_versions.go

```diff
@@ -566,6 +566,17 @@ const (
 	// push request and leaving the push outcome to the server-side logic.
 	V23_2_RemoveLockTableWaiterTouchPush
 
+	// V23_2_LeaseToSessionCreation lease table will have sessionID inside it.
```

Comment:
nit: Can we make these comments read like a full sentence like the rest of this file?

```suggestion
	// V23_2_LeaseToSessionCreation guarantees that the lease table will have a sessionID column inside of it.
```